### PR TITLE
Translate "Ruby 3.1.6 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translate #3260
Actual diff 6e7590b

### CODE REVIEW

File: ko/news/_posts/2024-05-29-ruby-3-1-6-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: "Ruby 3.1.6 릴리스"
+author: "hsbt"
+translator: "shia"
+date: 2024-05-29 9:00:00 +0000
+lang: ko
+---
+
+Ruby 3.1.6이 릴리스되었습니다.
+
+Ruby 3.1 버전대는 이제 보안 유지보수 단계에 들어갔습니다. 일반적으로 이 단계에서는 보안 문제만 수정합니다. 그러나 Ruby 3.1.5 릴리스 이후 빌드 실패 문제가 여럿 발생했습니다. 이 문제들을 해결하기 위해 Ruby 3.1.6을 릴리스하기로 결정했습니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-29-ruby-3-1-6-released.md:12]:
```suggestion
Ruby 3.1 버전대는 지금 보안 유지보수 단계입니다. 일반적으로 이 단계에서는 보안 문제만 수정합니다. 그러나 Ruby 3.1.5 릴리스 이후 빌드 실패 문제가 여럿 발생했습니다. 이 문제들을 해결하기 위해 Ruby 3.1.6을 릴리스하기로 결정했습니다.
```

File: ko/news/_posts/2024-05-29-ruby-3-1-6-released.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+아래 항목에서 자세한 내용을 확인하세요.
+
+* [Bug #20151: FreeBSD 14.0에서 Ruby 3.1을 빌드할 수 없음](https://bugs.ruby-lang.org/issues/20151)
+* [Bug #20451: 좋지 않은 Ruby 3.1.5 백포트로 fiddle 빌드에 실패함](https://bugs.ruby-lang.org/issues/20451)
```
Comment (yous) [inline ko/news/_posts/2024-05-29-ruby-3-1-6-released.md:17]:
```suggestion
* [Bug #20451: 잘못된 Ruby 3.1.5 백포트로 fiddle 빌드에 실패함](https://bugs.ruby-lang.org/issues/20451)
```

File: ko/news/_posts/2024-05-29-ruby-3-1-6-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-29-ruby-3-1-6-released.md]: fixed e065bd8

File: ko/news/_posts/2024-05-29-ruby-3-1-6-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-29-ruby-3-1-6-released.md]: fixed e065bd8

**Review by yous — APPROVED** [review]
Thanks!

**Pull request merged**